x-pack/plugins/ml/public/file_datavisualizer/components/import_errors/errors.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiCallOut,
+  EuiAccordion,
+} from '@elastic/eui';
+
+import { IMPORT_STATUS } from '../import_progress';
+
+export function ImportErrors({ errors, statuses }) {
+  return (
+    <EuiCallOut
+      title={title(statuses)}
+      color="danger"
+      iconType="cross"
+    >
+      {
+        errors.map((e, i) => (
+          <Error error={e} key={i} />
+        ))
+      }
+
+    </EuiCallOut>
+  );
+}
+
+function title(statuses) {
+  switch (IMPORT_STATUS.FAILED) {
+    case statuses.readStatus:
+      return 'Error reading file';
+    case statuses.indexCreatedStatus:
+      return 'Error creating index';
+    case statuses.ingestPipelineCreatedStatus:
+      return 'Error creating ingest pipeline';
+    case statuses.uploadStatus:
+      return 'Error uploading data';
+    case statuses.indexPatternCreatedStatus:
+      return 'Error creating index pattern';
+    default:
+      return 'Error';
+  }
+}
+
+function Error(error, key) {
```

Comment:
Since `Error` is native to JavaScript maybe better to rename this so something like `ErrorAccordion`.

Reply:
good point!